### DISCUSSION TITLE
How to add lovelace cards in a floorplan?

### DISCUSSION THREAD

**Discussion by Tomasvdr198** [ExperienceLovelace/ha-floorplan#85] [Q&A]
Hi, 
I've been working a bit with ha floorplan and I really love it so far! But I was wondering if it is possible to place another lovelace card in an existing floorplan card? I want to add a media control with what is playing and stuff in the blue square on the right.

![image](https://user-images.githubusercontent.com/19536793/114450277-1bfffd00-9bd6-11eb-8f65-b803a7372b82.png)

I hope you can help me!

**Comment by exetico** [comment]
You could add media-buttons in your SVG, and bind service-calls to a media-player service or so, to control music. There's no way to add a normal element within a floorplan, if you think of a native element like a media-control element or so:
![image](https://user-images.githubusercontent.com/3549445/114569743-85831880-9c75-11eb-964d-bcf155b577f9.png)

I'm not even sure if it's possible to do, by modding ha-floorplan to support it. It will most likely take a good amount of time - but I'm not that sure.

Another take would be to use [browser-mod](https://github.com/thomasloven/hass-browser_mod), and use that to open the media-control in a popup :-)
